amethyst_ui/src/font/systemfont.rs

```diff
@@ -0,0 +1,32 @@
+use font_kit::handle::Handle;
+use font_kit::error::SelectionError;
+use font_kit::source::SystemSource;
+use font_kit::family_name::FamilyName;
+use font_kit::properties::{Properties, Style};
+
+// Temporarily disabled because const can't contain heap allocations.
+// Fantasy contains comic sans btw
+//const DEFAULT_FONTS: &[FamilyName] = &[FamilyName::Title("arial".to_string()), FamilyName::SansSerif, FamilyName::Monospace, FamilyName::Fantasy];
```

Comment:
Not interfering with the merge, but I'd recommend PRing Arial as a default family name to that crate you use.

Reply:
I would actually PR a way to declare FamilyName without an allocation if possible

Reply:
Should remove this, const will never have the ability to contain heap allocations.